src/Symfony/Component/Lock/Tests/Store/FlockStoreTest.php

```diff
@@ -46,14 +46,23 @@ public function testConstructWhenRepositoryDoesNotExist()
     public function testConstructWhenRepositoryIsNotWriteable()
     {
         $this->expectException('Symfony\Component\Lock\Exception\InvalidArgumentException');
-        $this->expectExceptionMessage('The directory "/" is not writable.');
+        $this->expectExceptionMessage('The FlockStore directory "/" is not writable.');
         if (!getenv('USER') || 'root' === getenv('USER')) {
             $this->markTestSkipped('This test will fail if run under superuser');
         }
 
         new FlockStore('/');
     }
 
+    public function testConstructWithSubdir()
+    {
+        if (!getenv('USER') || 'root' === getenv('USER')) {
+            $this->markTestSkipped('This test will fail if run under superuser');
```

Comment:
is this actually true ? It does not rely on some directories not being writable.

Reply:
good catch. fixed by https://github.com/symfony/symfony/pull/39721